.github/workflows/QA.yaml

```
@@ -70,3 +70,19 @@ jobs:
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v4
+
+      - name: Install
+        uses: ./.github/common-actions/install
+
+      - name: Build packages
+        run: pnpm build
+
+      - name: Release
+        run: pnpx pkg-pr-new publish --compact --pnpm './packages/**/*'
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)